CHANGELOG.md

```diff
@@ -9,13 +9,68 @@ This is a high level and scarce summary of the changes between releases.
 We would recommend to consult log of the 
 [DataLad git repository](http://github.com/datalad/datalad) for more details.
 
-## 0.10.4 (??? ??, 2018) -- Soon-to-be-perfect
+## 0.10.4 (Oct 23, 2018) -- Soon-to-be-perfect
 
 ### Major refactoring and deprecations
 
 - `datalad.consts.LOCAL_CENTRAL_PATH` constant was deprecated in favor
   of `datalad.locations.default-dataset` [configuration] variable
-  (#2835)
+  ([#2835])
+
+### Minor refactoring
+
+- `"notneeded"` messages are no longer reported by default results
+  renderer
+- [run] no longer shows commit instructions upon command failure when
+  `explicit` is true and no outputs are specified ([#2922])
+- `get_git_dir` moved into GitRepo ([#2886])
+- `_gitpy_custom_call` removed from GitRepo ([#2894])
+- Eliminated majority of uses of GitPython's `.repo.rev_parse` by
+  adding `GitRepo.format_commit` ([#2902])
```

Comment:
The new `.format_commit` method deserves mention, but I view the `.repo.rev_parse` bit as an internal implementation detail that just adds noise.

Reply:
indeed, feel welcome to tune up more to your liking! I trust ya!